scripts/build

```diff
@@ -16,5 +16,5 @@ if [ "$CROSS" = "true" ] && [ "$ARCH" = "amd64" ]; then
     GOOS=windows go build -ldflags "$LINKFLAGS" -o bin/support-bundle-utils-windows
 fi
 
-
-
+# non-go scripts
+cp bin_scripts/* bin
```

Comment:
let's rename the `bin_scripts` dir to `hack` instead.

Reply:
fixed!